source/install/cloud-changelog.md

```diff
@@ -4,12 +4,43 @@ This changelog summarizes updates to [Mattermost Cloud](https://mattermost.com/g
 
 Latest Mattermost Cloud releases:
 
+- [Release 2023-04-13](#release-2023-04-13)
 - [Release 2023-03-29](#release-2023-03-29)
 - [Release 2023-03-20](#release-2023-03-20)
 - [Release 2023-01-26](#release-2023-01-26)
 - [Release 2023-01-16](#release-2023-01-16)
 - [Release 2022-12-20](#release-2022-12-20)
-- [Release 2022-12-01](#release-2022-12-01)
+
+## Release 2023-04-13
+
+### Improvements
+
+#### User Interface (UI)
+ - Enabled Work Templates in Onboarding checklist and `+` Menu.
+ - Added a marketplace button to the bottom of the apps bar. The button is visible if the marketplace is enabled and the user has ``SYSCONSOLE_WRITE_PLUGINS`` permissions.
+ - Added a "Add channels" button to the bottom of the left-hand sidebar to make it more obvious for users on how to create or join channels.
+ - Removed Webapp Build Hash since it is now identical to Server Build Hash.
```

Comment:
Can we specify where the Webapp Build Hash details have been removed please? (I believe it's the About modal, correct?)